deploy/2_create_app_namespace.sh

```diff
@@ -3,27 +3,25 @@ set -euo pipefail
 
 . utils.sh
 
-announce "Creating Test App namespace."
+announce "Creating Application namespace."
 
-if [[ $PLATFORM == openshift ]]; then
+if [[ $PLATFORM == openshift && "${DEV}" = "false" ]]; then
   $cli_with_timeout "login -u $OPENSHIFT_USERNAME"
 fi
 
-set_namespace default
```

Comment:
don't we need this?

Reply:
it stales the local dev so wrapping it in conditional

Reply:
negative, seems a bit repetitive here since we are just hopping between namespaces